src/server/server.rs

```diff
@@ -33,6 +33,7 @@ use super::{Config, Result};
 const LOAD_STATISTICS_SLOTS: usize = 4;
 const LOAD_STATISTICS_INTERVAL: Duration = Duration::from_millis(100);
 pub const GRPC_THREAD_PREFIX: &str = "grpc-server";
+pub const READPOOL_NORMAL_THREAD_PREFIX: &str = "store-read-norm";
```

Comment:
I think it's reasonable to leave out high priority requests since they are more sensitive to latency. As for low-prio, we will add a seperate `Batcher` to make it work. Maybe only filter normal-priority requests?

Reply:
Is point get normal priority?